config/config.js

```diff
@@ -0,0 +1,33 @@
+import env from "../config/env.generated.json";
```

Comment:
I'd still recommend renaming this to `user-data.js`, because the thing this file exports is not a configuration object at all.